src/gps/gpsd_test.cpp

```diff
@@ -0,0 +1,136 @@
+/* Note from Nathan Hatch (Husky Robotics):
+ *
+ * This code is copied from https://github.com/jcable/gpsd/blob/master/test_gpsmm.cpp
+ * and modified slightly. (There was a change in how the gps_data_t struct stores
+ * the information about the satellites.)
+ */
+
+/*
+ * Copyright (C) 2010 Eric S. Raymond.
+ *
+ * This software is distributed under a BSD-style license. See the
+ * file "COPYING" in the top-level directory of the distribution for details.
+ *
+ */
+
+/* This simple program shows the basic functionality of the C++ wrapper class */
+#include <iostream>
+
+#include "libgpsmm.h"
+
+using namespace std;
+
+/* 
+ * We should get libgps_dump_state() from the client library, but
+ * scons has a bug; we can't get it to add -lgps to the link line,
+ * apparently because it doesn't honor parse_flags on a Program()
+ * build of a C++ file.
+ */
+static void libgps_dump_state(struct gps_data_t *collect)
+{
+    /* no need to dump the entire state, this is a sanity check */
+#ifndef USE_QT
+    /* will fail on a 32-bit macine */
+    (void)fprintf(stdout, "flags: (0x%04x) %s\n",
```

Comment:
Is there a reason these use `fprintf(stdout, ...)` instead of just `printf`? Not saying it has to change, just curious.